frontend/src/modules/editor/components/Editor.js

```diff
@@ -163,25 +163,27 @@ export class EditorBase extends React.Component<InternalProps> {
             return null;
         }
 
-        return <div className="editor">
+        return <React.Fragment>
```

Comment:
I don't like that, it makes this component not encapsulated. I'd rather keep the `editor` class at the top, and add another class to the underlying editor, rather than doing this. That also conflicts a lot with my current refactoring, and the code I have will make this easier to do because I've renamed a few components. If you could keep the changes to a minimum here that would be very helpful to me. :angel:

Reply:
OK. In this case the menu will have to break slightly out of boundaries on narrower screens and we'll postpone making it prettier after your refactor or even until we make the UI responsive.